x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat_ecs/ml/rare_process_linux_ecs.json

```diff
@@ -0,0 +1,49 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "SIEM Auditbeat: Detect unusually rare processes on Linux (beta)",
+  "groups": ["siem"],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare process executions on linux",
```

Comment:
for `Linux` we use lowercase `l` in the detector and an uppercase `L` in the job description. Let's use uppercase for both.

Reply:
Done in d33a13304d9b8cb3b6555dc7b29711b7319b7f57